### PR TITLE
update json-schema/draft4 test suite

### PR DESCRIPTION
**Pull request by ChALkeR**
Also use the full test suite + explicitly exclude failing, instead of deleting unsupported tests from the test suite.

Now the test runner has an explicit blacklist of unsupported tests, which can consist of files, blocks, or individual tests.

Tests taken from https://github.com/json-schema-org/JSON-Schema-Test-Suite (draft 4 for now).

### CODE REVIEW

File: test/json-schema.js [hunk 1]
```diff
@@ -3,26 +3,57 @@ const fs = require('fs')
...
+  'refRemote.json',
+  'ref.json',
+  // Blocks
+  'format.json/validation of IP addresses',
+  'format.json/validation of IPv6 addresses',
+  'items.json/items and subitems',
+  // Specific tests
+  'items.json/an array of schemas for items/JavaScript pseudo-array is valid',
+  'maxLength.json/maxLength validation/two supplementary Unicode code points is long enough',
+  'minLength.json/minLength validation/one supplementary Unicode code point is not long enough',
```
Comment (ChALkeR) [inline test/json-schema.js]: The two `maxLength/minLength` ones are fixed in https://github.com/ExodusMovement/is-my-json-valid/pull/22, so those two lines could be removed once that lands.

Comment (kklash) [reply test/json-schema.js]: This one is merged now, so these tests can be reapplied

Comment (ChALkeR) [reply test/json-schema.js]: Done, thanks! Rebased.

**Review by kklash — APPROVED** [review]
tACK

**Pull request merged**